tabuckner/broskichat
# Added simple login component

### Project Changed


**- What I Did**
Added simple login component

**- How I Did It**
Added simple login component
Added a thin wrapper around a bulma input and created a shared components directory. 
Lets try to continue this pattern to cut down on our HTML markup if we can.

**- How to Test/Verify It**
launch app
go to login page
enter email and pass
hit submit
check console

![image](https://user-images.githubusercontent.com/32392635/44375235-1949ba00-a4b8-11e8-8a1b-fd932f3f9c4d.png)

**- An Image to Make Us Giggle (not mandatory but encouraged)**
